ffwd-http-reporter/src/main/java/com/spotify/metrics/ffwdhttp/FastForwardHttpReporter.java

```diff
@@ -86,14 +88,15 @@ public Thread newThread(Runnable runnable) {
     private final Set<Percentile> histogramPercentiles;
     private final Clock clock;
     private final TagExtractor tagExtractor;
+    private final ResourceExtractor resourceExtractor;
 
     private final AtomicBoolean running = new AtomicBoolean(false);
     private ScheduledFuture<?> scheduledFuture;
 
     private FastForwardHttpReporter(
```

Comment:
This is HTTP version of reporter which is not used in Apollo. It is configured but not used.
The change here should correspond to the change in FFWD, actually in both agent version and metrics api.
Overall I disagree with this strategy - introducing resource tags will shoot us in the foot in the future.
I'd like to sync up about that strategy before we introduce those to overall pipeline.

Reply:
So I made these changes align with these changes: https://github.com/spotify/ffwd-http-client/pull/2

if we ever want to upgrade the ffwd-http-client in semantic metrics we'll need to support these changes / resource identifiers.